src/EnergyPlus/SetPointManager.cc

```diff
@@ -4707,7 +4723,17 @@ namespace SetPointManager {
 			SetTempAtOutHigh = this->OutHighSetPt1;
 		}
 
-		SetPt = CalcSetPoint(OutLowTemp, OutHighTemp, OutDryBulbTemp, SetTempAtOutLow, SetTempAtOutHigh);
+		if ( CtrlType == iCtrlVarType_MaxTemp ) {
+			SetPt = SetTempAtOutLow;
+			this->SetPtHi = SetPt;
+		} else if ( CtrlType == iCtrlVarType_MinTemp ) {
+			SetPt = SetTempAtOutHigh;
+			this->SetPtLo = SetPt;
+		} else {
+			SetPt = CalcSetPoint( OutLowTemp, OutHighTemp, OutDryBulbTemp, SetTempAtOutLow, SetTempAtOutHigh );
+			this->SetPtLo = SetTempAtOutHigh;
+			this->SetPtHi = SetTempAtOutLow;
+		}
```

Comment:
No need to introduce new variables SetPtLo and SetPtHi.  Just use SetPt.   This manager only needs to set one value on the node.  A dual-setpoint plant loop would require two setpoint managers, one for the high setpoint and one for the low.